### DISCUSSION TITLE
applying filters with single column and all columns results are changing

### DISCUSSION THREAD

**Discussion by vihariazure** [milvus-io/milvus#38689] [Q&A and General discussion]
hi team
i applied filter as 'price >2000' and output fields as ['price']
below are the results

![image](https://github.com/user-attachments/assets/25c108f1-daaf-4522-8281-55446d3eb9f3)

now with the same filter and output fields as ['*']
below are the results
![image](https://github.com/user-attachments/assets/53250b23-320a-4d5a-b857-d4018ce0727f)

with price >2000 and output as all fields the price is column values are less then 2000 as you can use above attachment.
what is the reason for that results . how to fix it

**Comment by yhmo** [comment]
Possible reason: there are duplicate ids in the collection.
Did you insert two items with the same id 837291000? One item price is 2699, the other is 44.99?

**Reply by vihariazure** [reply]
thanks for the reply i will check and update
